packages/core-data/src/resolvers.js

```diff
@@ -749,12 +749,11 @@ export const getNavigationFallbackId =
 
 export const getDefaultTemplateId =
 	( query ) =>
-	async ( { dispatch, registry } ) => {
-		const response = await apiFetch( {
+	async ( { dispatch, registry, resolveSelect } ) => {
+		const template = await apiFetch( {
 			path: addQueryArgs( '/wp/v2/templates/lookup', query ),
-			parse: false,
 		} );
-		const template = await response.json();
+		await resolveSelect.getEntitiesConfig( 'postType' );
```

Comment:
Let's add an inline comment here. Otherwise, the resolution seems out of place and can be accidentally removed.